core/src/replay_stage.rs

```diff
@@ -2400,6 +2465,20 @@ impl ReplayStage {
         let new_root = tower.record_bank_vote(bank);
 
         if let Some(new_root) = new_root {
+            if first_alpenglow_slot.is_none() {
+                if let Some(activation_slot) = bank_forks
+                    .read()
+                    .unwrap()
+                    .root_bank()
+                    .compute_pending_activation_slot(&agave_feature_set::alpenglow::id())
+                {
+                    *first_alpenglow_slot = Some(activation_slot);
```

Comment:
I believe I know the answer but just to confirm, we'll only use `>=` to compare against this slot and NOT `==` right ? Ie, fi the first leader of the new epoch is offline or something

Reply:
Yep that's correct, if you check out the 2 consumers `process_replay_results` and `maybe_start_leader`, they both use >=